detox/webhook_server.js

```diff
@@ -7,8 +7,8 @@ const axios = require('axios');
 const ClientOAuth2 = require('client-oauth2');
 const express = require('express');
 
-const postMessageAs = require('./e2e/plugins/post_message_as');
-const webhookUtils = require('./e2e/utils/webhook_utils');
+const postMessageAs = require('./utils/post_message_as');
```

Comment:
Keeping this in js for now as it's a script to bring up a webhook server and outside e2e folder scope